go/engine/identify2_with_uid.go

```diff
@@ -399,6 +398,7 @@ func (e *Identify2WithUID) run(m libkb.MetaContext) {
 			m.Debug("| error during cancel: %+v", err)
 		}
 	}
+	e.unblock(m, true /* isFinal */, err)
```

Comment:
I'm reasonably sure the only downside of this is to very slightly slow the caller down. But every-so-slightly, I doubt we'll ever notice it.